publish-to-gh-pages.sh

```diff
@@ -21,6 +21,13 @@ git clone --single-branch --branch gh-pages git@github.com:adobe/hugo-spectrum.g
 rm -rf site-pages/*
 
 hugo -v --source="$PWD/exampleSite" --destination="$PWD/site-pages" --ignoreCache
+docker run --rm \
```

Comment:
I think the previous line can be safely deleted.

Reply:
Good catch :)